packs/nest/actions/humidity.py

```diff
@@ -0,0 +1,14 @@
+from lib import actions
+
+class HumidityAction(actions.BaseAction):
+    def run(self, structure, device, target=None):
+        if target:
+            target_humidity = target
+        else:
+            target_humidity = self._get_device(structure, device).humidity
```

Comment:
@Kami but whyyyyyy? I cannot think of a compelling reason to go one way or another right now, other than simpler actions. That may be reason enough, but idk.